unity-renderer/Assets/Scripts/MainScripts/DCL/Analytics/SocialAnalytics/SocialAnalyticsTestHelpers/SocialAnalyticsTestHelpers.cs

```diff
@@ -0,0 +1,36 @@
+using DCL.Interface;
+using NSubstitute;
+
+namespace SocialFeaturesAnalytics.TestHelpers
+{
+    public class SocialAnalyticsTestHelpers
+    {
+        public static ISocialAnalytics CreateMockedSocialAnalytics()
+        {
+            ISocialAnalytics mockedSocialAnalytics = Substitute.For<ISocialAnalytics>();
+
+            mockedSocialAnalytics.When(x => x.SendPlayerMuted(Arg.Any<string>())).Do(x => { });
```

Comment:
i think its not necessary to stub all this methods, since they do `void` returns

Reply:
Really? I didn't know that. If it is so, I could remove directly all my `SocialAnalyticsTestHelpers` class that I was used only to mock the functions in the tests.

Reply:
You were right! Done in https://github.com/decentraland/unity-renderer/pull/2257/commits/d30856f5f3282275be46a58fee5acf1e9102b83e